sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -20,18 +20,18 @@ package org.apache.spark.sql.hive.client
 import java.io.{File, PrintStream}
 import java.lang.reflect.InvocationTargetException
 import java.net.{URL, URLClassLoader}
-import java.util
 
+import java.util
```

Comment:
? This change still looks weird.

Reply:
Thanks, now `./dev/scalastyle` should pass.